### ISSUE TITLE
add vnic details property in an example packer json template

### ISSUE THREAD

**Issue by andreww35t** [oracle-quickstart/oci-quickstart-template#9]
Would it be possible to improve the packer template so that it contains the [create_vnic_details](https://www.packer.io/docs/builders/oracle/oci#create_vnic_details) property?  The example and doc listed at https://www.packer.io/docs/builders/oracle/oci#create_vnic_details  has not worked.  I slacked @benofben  about this.  I will follow up on any potential bug or support elsewhere, but I wanted to possibly help improve the examples.  If I get it working, I am happy to submit a pr.   TA.

p.s. apologies in advance if this constitutes some etiquette failure.

**Comment by zkaufman** [comment]
@andreww35t Yes please feel free to submit a PR with which contains the create_vnic_details and I will be happy to review and accept (if it works ;)).

Please note that we frequently provide the Quick Start template to partners to facilitate them getting their images and stacks deployed into OCI and OCI Marketplace. For that reason we ask that your code submission err on the side of simplicity and readability whenever reasonable. Thank for the suggested improvement and contributing!

**Comment by lucassrg** [comment]
Hi @andreww35t, can you please create an issue under the [Packer repo](https://github.com/hashicorp/packer/issues) since this is an issue very specific to Packer documentation for OCI and not a quickstart related issue.
 
Please provide the details and error messages so that we can work on identifying the root cause of the issue and provide you a solution.

Thanks,
Lucas
